### DISCUSSION TITLE
Github Achievements not yet calculated as expected, is it ok?

### DISCUSSION THREAD

**Discussion by orchestr7** [community/community#18242] [Other Feature Feedback, Questions, & Ideas]
<img width="1030" alt="image" src="https://user-images.githubusercontent.com/58667063/173033171-aba22341-f7b1-4d1f-87ab-309e091f2bce.png">

Some badges does not appear: for example for answering discussions. 
Probably we need some clear description of what to do to get a badge.

I am not complaining about that, but would like to help this great idea to improve and grow. I suppose you have some small bug in data calculation.

PS This feature is awesome! ❤️

**Comment by atulkamble** [comment]
In my case, badges are not reflecting yet.

**Reply by pecraveiro** [reply]
Same here, they're showing on the mobile app, but at the website I can't see...

**Comment by Yourun-proger** [comment]
Hello! I have the same problem. I got a "Pull Shark" showing that I only have 2 PRs...

**Comment by 1-ARIjitS** [comment]
i have the same issue has anybody got any solutions???? please reply

**Comment by radar-blip** [comment]
Info: [Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)

**Comment by mgnat16** [comment]
> <img alt="image" width="1030" src="https://user-images.githubusercontent.com/58667063/173033171-aba22341-f7b1-4d1f-87ab-309e091f2bce.png">
> 
> I have very strange badges that have taken random PR and showing that I have only 3 PRs :D Also other badges does not appear: for example for answering discussions. Probably we need some clear description of what to do to get a badge.
> 
> I am not complaining about that, but would like to help this great idea to improve and grow. I suppose you have some small bug in data calculation.
> 
> PS This feature is awesome! ❤️

+++++++++++

**Comment by rashidwassan** [comment]
How to add that '5 Discussions Answered' section?🥹

**Comment by ghost** [comment]
See pull  shark  achievementx1 Means you have did 2 pull requests  and pull shark achievementx2 Means you have pulled  16 pull  requests
![Screenshot_2023-01-15-23-46-39-151_com android chrome](https://user-images.githubusercontent.com/121353576/212559413-798b380d-17fe-476e-b1ec-48938e7d374b.jpg)
